Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build -a -o namespace-scope-opera
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 # FROM gcr.io/distroless/static:nonroot
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3-230
```

Comment:
@bitscuit 
Ask for curiosity, I found the `openssl` issue was in version `1.1.1g-11.el8`, and it is fixed in `1.1.1g-12.el8_3`.

and the iam-operator uses the digest in `8.3-230` to fix `openssl` issue. https://github.com/IBM/ibm-iam-operator/pull/277

May I ask how do we know that the `1.1.1g-12.el8_3` version is equal to the tag `8.3-230`?

Reply:
Generally, issues with packages are resolved by just updating to newer version of the package, and the packages are from the base image. So we update the base image to the latest available version to pick up the latest packages. At this time, `8.3-230` is the latest ubi8 image.